lib/ably/rest/channel.rb

```diff
@@ -65,17 +64,25 @@ def initialize(client, name, channel_options = {})
       #   ]
       #   channel.publish messages
       #
-      def publish(name, data = nil, attributes = {})
-        messages = if name.kind_of?(Enumerable)
-          name
+      #   # Publish a single Ably::Models::Message object, with a query params
+      #   # specifying quickAck: true
+      #   message = Ably::Models::Message(name: 'click', { x: 1, y: 2 })
+      #   channel.publish message, {quickAck: 'true'}
```

Comment:
Pedantic, but `channel.publish message, quickAck: 'true'` is more Ruby-esque
If you must have `{}` then this lib has stuck with spaces i.e. `channel.publish message, { quickAck: 'true' }`